test/gtest/uct/test_sockaddr.cc

```diff
@@ -755,9 +810,16 @@ class test_uct_sockaddr : public uct_test {
     std::vector<char>      m_long_priv_data;
     pthread_mutex_t        m_ep_client_data_lock;
     ep_client_data_map_t   m_ep_client_data;
+
+    static std::atomic<bool>           m_close_listener_notify;
+    static std::atomic<uct_listener_h> m_server_uct_listener;
```

Comment:
but these are still here... My point was if we can do the same with existing members, but probaly convert counters to atomics

Reply:
unfortunately we can't, because these static members are used to synchronize new callback and thread